ui/webui/src/components/installation/InstallationProgress.jsx

```diff
@@ -59,6 +58,25 @@ export class InstallationProgress extends React.Component {
                                             this.props.onAddErrorNotification
                                         );
                             }
+                            console.log("MESSAGE");
+                            console.log(step);
+                            console.log(message);
+                            // storage
+                            if (step <= 3) {
+                                this.setState({ currentProgressStep: 0 });
+                            }
+                            // payload
+                            if (step === 4) {
```

Comment:
You 'd rather use else if.

In this way if won't go over all conditions, if the first is truthy,